CONTRIBUTING.md

```diff
@@ -26,7 +26,7 @@ be accepted under the terms of that license.
 Checkstyle failures during compilation indicate errors in your style and will
 be displayed in the console output of the build (including in Travis-CI output),
 or can be viewed in the `checkstyle-result.xml` file.  To save yourself some
-hassle, consider executing checkstyle first per: 'mvn checkstyle:checkstyle'.
+hassle, consider executing checkstyle first per: `mvn checkstyle:checkstyle`.
```

Comment:
This too: given the absence of maven-checkstyle-plugin in the POM, the `checkstyle.xml` file wouldn't be used for `checkstyle:checkstyle` (default is the bundled `sun_checks.xml`: https://maven.apache.org/plugins/maven-checkstyle-plugin/checkstyle-mojo.html#configLocation), so it's likely that checkstyle would fail:

Currently at HEAD:
```
[INFO] There are 1627 errors reported by Checkstyle 8.29 with sun_checks.xml ruleset.
```

 And checkstyle is not run in CI either.

Reply:
This was addresses with #234.